src/components/Profile/Profile.scss

```diff
@@ -4,4 +4,14 @@
     button {
         margin-bottom: 1rem;
     }
+    .success {
+        display: flex;
+        align-items: center;
+        font-weight: 600;
```

Comment:
create a variable for font-weight: 600, it's used in a lot of places

`$fontweight-semibold: 600`